.circleci/config.yml

```diff
@@ -660,9 +652,7 @@ jobs:
             dpkg -i packages-microsoft-prod.deb
             apt-get install -y apt-transport-https
             apt-get update
-            apt-get install -y dotnet-sdk-3.1
-            mkdir -p ~/.nuget/{NuGet,packages}
-            cp build/NuGet.Config ~/.nuget/NuGet/
+            apt-get install -y dotnet-sdk-3.1 git
```

Comment:
why is `git` needed in this step now?

Reply:
we use a `git clean -xfd` command in our C# clean scripts.